webapp/src/ts/main.ts

```diff
@@ -22,6 +22,10 @@ import { POUCHDB_OPTIONS } from './constants';
 
 import * as bootstrapper from '../js/bootstrapper';
 
```

Comment:
In the original code we have more locales: https://github.com/medic/cht-core/blob/9696b810fbad8c9b8a287bd97139a367f241c56e/webapp/src/js/app.js#L39
But I tested and looks like it's loading the languages without  needing to add `require().`

Reply:
yes, they're being loaded by default. I've actually added a webpack script that we pass specific languages to load, because otherwise all would be loaded. 
https://github.com/medic/cht-core/blob/4750-angular-10-take-2/webapp/custom-webpack.config.js#L31